Assets/HotmaxEditorVR/activeProp.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class activeProp : MonoBehaviour
+{
+
+    //--local refs
+    private GameObject _selectedObject = stateManager.selectedObject;
+    private bool _selectedObjectIsActive = stateManager.selectedObjectIsActive;
+    private stateManager.editorModes _editorMode = stateManager.editorMode;
+
+    //--put the monobehaviours into a list
+    private List<MonoBehaviour> decorators = new List<MonoBehaviour>();
```

Comment:
change to `Dictionary<String, MonoBehavior>`
on stateMachine initial loop change to:
```
foreach (Monobehavior decorator in decorators.Values)
{
   decorator.enabled = false;
}
```
and add to decorators using string key, and reference them in the state machine with that string key

Reply:
you spelled MonoBehaviour wrong, but... got it